examples/27.ammDeposit.js

```diff
@@ -46,30 +40,36 @@ const dvfConfig = {
 
   const waitForDepositCreditedOnChain = require('./helpers/waitForDepositCreditedOnChain')
 
-  if (process.env.DEPOSIT_FIRST === 'true') {
```

Comment:
this was automatically removed since the file was out of sync with the template. this suggests it was manually editted. I can add this change to the template instead so the example is generated correctly